### ISSUE TITLE
Nerdamer + Observable notebook

### ISSUE THREAD

**Issue by yosuke** [jiggzson/nerdamer#420]
Just wanted to share that using nerdamer with observable notebook was possible with very nice user experience:

https://beta.observablehq.com/@yosuke/symbolic-math-using-nerdamer

Only the problem was, it required a bit of twist to import nerdamer core module (had to use bundle.run service) probably due to the way how nerdamer export the module:

https://beta.observablehq.com/@yosuke/nerdamer-all

Tricks I used to import the modules are described here:

https://beta.observablehq.com/@tmcw/requiring-modules-troubleshooting

**Comment by jiggzson** [comment]
@yosuke, looks nice. Thanks for bringing this issue to my attention. I'll have to dig into `bundle.run` a bit more. I was under the impression that checking for `module` and then acting accordingly was enough but that might not be the case. I'm guessing that `require('nerdamer/all')` didn't work as outlined in issue #113?

**Comment by jiggzson** [comment]
I see. I've been meaning to break up the files and create bundles for some time but I've just been putting off. That would be an opportune time to revisit this issue. Issue number 3 does raise some concern. Are you saying that symbolic evaluation doesn't work when importing in that manner?

**Comment by yosuke** [comment]
@jiggzson Here is the example notebook to reproduce the issue:

https://beta.observablehq.com/@yosuke/test-to-reproduce-nerdamer-issue-420

**Comment by yosuke** [comment]
Seems they are using d3-require to realize their require function:

https://github.com/observablehq/stdlib/blob/master/src/require.js

https://github.com/d3/d3-require

So the problem may be specific to this lib.

**Comment by yosuke** [comment]
Ahh, it was written in the d3-require README

```
Only the built-in exports dependency is allowed; no require or module as in CommonJS.
```

Understand that is why...

**Comment by yosuke** [comment]
@jiggzson Observable notebook is using d3-require library internally to realize require function and d3-require library does not support commonjs style module export. “Module” detection of nerdamer will not work on this environment.

Also require(’nerdamer/all’) may work if it is implemented in ES6 style export.

**Comment by jiggzson** [comment]
@yosuke, thanks for the clarification and investigative work. I'll keep this in mind going forward.

**Comment by jiggzson** [comment]
@yosuke, I apologize for the long delay on this issue. I was putting it off because I was going to do a major refactor but time constraints have made this impossible. I decided to proceed by just adding the file for now.

**Comment by jiggzson** [comment]
Fixed as of version 1.1.11.
